File: Algorithms/src/1980. Find Unique Binary String/Code/solution.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        int n = nums.length;
        StringBuilder result = new StringBuilder();

        // Flip the diagonal bits
        for (int i = 0; i < n; i++) {
            if (nums[i].charAt(i) == '0')
                result.append('1');
            else
                result.append('0');
        }

        return result.toString();
    }
}

File: Algorithms/src/1980. Find Unique Binary String/Code/solution.js
/**
 * @param {string[]} nums
 * @return {string}
 */
var findDifferentBinaryString = function (nums) {
  let n = nums.length;
  let result = "";

  // Flip the diagonal bits
  for (let i = 0; i < n; i++) {
    result += nums[i][i] === "0" ? "1" : "0";
  }

  return result;
};
